vignettes/advanced.Rmd

```diff
@@ -26,102 +29,290 @@ knitr::opts_chunk$set(
 The purpose of this vignette is to provide an overview of some of the more advanced
 features of the `rbmi` package. 
 
+# Data simulation {#sec:dataSimul}
+
+In order to demonstrate the advanced functions we will first create a simulated dataset with the `rbmi` function `simulate_data()`.
+A reader interested in other sections of the vignette must not necessarly read through this section.
+The `simulate_data()` function generates data for a two-arms clinical trial with longitudinal continuous outcome and two intercurrent events (ICEs).
+One intercurrent event (ICE1) may be thought of as a discontinuation from study treatment due to study drug or condition related (SDCR) reasons.
+The other one (ICE2) may be thought of as discontinuation from study treatment due to uninformative study drop-out.
+For the purposes of this vignette we consider the following scenario which is similar to the simulation study used in @Wolbers2021.
+It is a trial of an active drug versus placebo with 100 subjects per group and 6 post-baseline assessments (bi-monthly visits until 12 months):
```

Comment:
Perhaps reword, it just doesn't seem qutie right, at least the opener of "it is a trial" doesn't read quite right to me.

Reply:
"Let's consider a trial of ...."